docs/authentication.rst

```diff
@@ -24,21 +36,13 @@ required, or continue into your middleware::
       res.send('Hello, ' + req.user.fullname);
     });
 
-For browser-based clients, you can control the idle time and expiration time
-of the session by modifying the OAuth Policy for the Stormpath Application
-that you are using.
-
-To change these settings, you should invoke a node client directly::
+Behind the scenes we are using OAuth2 Access Tokens, and storing them in the
+cookies.  The lifetime of the cookies is controlled by the OAuth Policy of
+your Stormpath Application.  By default the cookies will expire after 60 days.
 
-    var stormpath = require('stormpath'); // Using the Node SDK directly
-    var client = new stormpath.Client();
-
-    client.getApplication('your app href',function (err,application){
-        application.getOAuthPolicy(function (err,policy){
-            policy.accessTokenTtl = "PT1D"; // one day
-            policy.save();
-        });
-    });
+If you need to change the expiration time of the cookies, please login to the
+Stormpath Admin Console and find the OAuth policy inside of your Stormpath
+Application.  Then change the expiration time of the Refresh Token.
```

Comment:
Why are we mentioning the refresh token expiration time here? We should explain it first, I think =)